packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -0,0 +1,50 @@
+import { IForwarder, IPrismInput } from '@stoplight/prism-core';
+import { IHttpOperation, IServer } from '@stoplight/types';
+import axios from 'axios';
+import { IHttpConfig, IHttpRequest, IHttpResponse } from '../types';
+
+export class HttpForwarder
+  implements IForwarder<IHttpOperation, IHttpRequest, IHttpConfig, IHttpResponse> {
+  public async forward(opts: {
+    resource?: IHttpOperation;
+    // todo: IHttpRequest should be enough
+    input: IPrismInput<IHttpRequest>;
+  }): Promise<IHttpResponse> {
+    if (!opts.resource) {
+      throw new Error('Missing spec');
+    }
+    if (!opts.resource.servers || opts.resource.servers.length === 0) {
+      throw new Error('Server list is missing in spec');
+    }
+
+    const response = await axios({
+      method: opts.input.data.method,
+      url: this.resolveServerUrl(opts.resource.servers[0]) + opts.input.data.url.path,
```

Comment:
Nothing wrong with `this.resolveServerUrl(opts.resource.servers[0])` but would be more generic if you passed `this.resolveServerUrl(opts.resource)` and let the function "decide" to choose first server.

Also `template strings` ;). Adding strings up is sooooo 2000 ;)

Reply:
Template strings are nice feature, but which one is prettier in that case: `${a}${b}` or `a + b` ?
I vote on the latter ;-)

Reply:
I agree with @StefanDywersant :). Minor, but I generally use concatenation if its a simple combination of vars, and string interpolation otherwise.